spec/system/formative_assessment_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'Formative assessment' do
+  include_context 'with progress'
+
+  before do
+    view_pages_before_formative_assessment(alpha)
+  end
+
+  include_context 'with user'
+
+  context 'when a user has visited each module up to and including a formative assessment' do
+    it 'can click to resume training to visit the formative assessment page' do
```

Comment:
This could instead provide clarity by saying something like:

`call to action links to the last visited page`